src/bin/pg_autoctl/cli_do_root.c

```diff
@@ -230,6 +230,27 @@ CommandLine do_pgsetup_commands =
 					 "Manage a local Postgres setup", NULL, NULL,
 					 NULL, do_pgsetup);
 
+CommandLine do_tmux_script =
+	make_command("script",
+				 "Produce a tmux script for a demo or a test case",
+				 "[option ...]",
+				 "  --root          path where to create a cluster\n" \
+				 "  --first-port    first Postgres port to use (5500)\n" \
```

Comment:
There's no makefile variable that allows you to set this. Thus disallowing you to spawn two different clusters at the same time.

Reply:
Fixed now, use FIRST_PGPORT from the Makefile.